README.md

```diff
@@ -31,7 +31,7 @@ Java 8+
 
 ### References <a name="references"></a>
 * [JWE Specification](https://datatracker.ietf.org/doc/html/rfc7516)<br/>
```

Comment:
Very minor:
* Maybe use the page title "JSON Web Encryption (JWE)" instead of "JWE Specification" 
* We should be able to remove the `<br/>` element now